.eslintrc

```diff
@@ -1,3 +1,11 @@
 {
-  "extends": "standard"
+  "extends": "standard",
+  "parserOptions": {
+    "sourceType": "script"
```

Comment:
I know this is necessary for the `strict` rule to work correctly, but I don't think it's accurate - most of the files are modules, right?